js/samples/06.auth.adaptiveCardSSO/src/index.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Import required packages
+import { config } from 'dotenv';
+import * as path from 'path';
+import * as restify from 'restify';
+
+// Import required bot services.
+// See https://aka.ms/bot-services to learn more about the different parts of a bot.
+import {
+    ActivityTypes,
+    CardFactory,
+    CloudAdapter,
+    ConfigurationBotFrameworkAuthentication,
+    ConfigurationBotFrameworkAuthenticationOptions,
+    MemoryStorage,
+    MessagingExtensionAttachment,
```

Comment:
I know you're going to hate this but unfortunately, `MessagingExtension(s)` is the incorrect terminology. `MessageExtension` is correct. Please do a rename (I asked the same for Chaoyi's PR). 

- if it's an import from botbuilder, it's fine to leave it as-is. We can't do anything about pre-existing incorrect naming.
- From here on out, any instance of message extension related names should be kept as MessageExtension, and Messaging... should be avoided as much as possible. 
- Same is applicable to samples

Sorry for not catching this earlier.